### DISCUSSION TITLE
add playlist indication

### DISCUSSION THREAD

**Discussion by SunsetTechuila** [nimsandu/spicetify-bloom#221] [Show and tell]
![image](https://user-images.githubusercontent.com/115353812/229557544-ca029772-42e2-4b76-a854-0afc8a3e1405.png)

https://github.com/SunsetTechuila/spicetify-bloom/commit/6d4b6a43b7c4ec1e55e37411430be064385e09b9

@nimsandu what do you think?

**Comment by nimsandu** [comment]
@SunsetTechuila You are testing a concept which makes the below playlist icon small than the above one so you can give the illusion of a 3D arrangement right?

**Reply by SunsetTechuila** [reply]
no, just making active a bit bigger

**Reply by nimsandu** [reply]
@SunsetTechuila I'm not so sure about this implementation. we are following Microsoft fluent design principles. so we are always trying to bring Spotify listening experience a bit closer to native windows 11 experience and this doesn't feel fine in my opinion. you and i decided to remove play icon which indicated the now playing playlist due to user confusions. so this doesn't feel necessary in my Pov. 

but remember this is my personal thoughts! any fact, point, feeling or analysis will be considered in our blooming community! so every fact, point, feeling or analysis will be warmly appreciated.

@SunsetTechuila thank you very much for your contributions to make our theme uniquely ours

**Reply by SunsetTechuila** [reply]
so, what do you suggest? do not indicate active playlist at all?

**Reply by nimsandu** [reply]
> so, what do you suggest? do not indicate active playlist at all?

umm, it seems fine for now ig

**Reply by SunsetTechuila** [reply]
Ok

**Reply by nimsandu** [reply]
thanks for your unmatching helps @SunsetTechuila
